packages/mediacenter/kodi/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="kodi"
-PKG_VERSION="bb0699db41d50a613de88f2bbe8124d4edf269d3"
-PKG_SHA256="28bd28a85e47995d4965ac402228374662bb85ba0644c94c52f371d97c87db1b"
+PKG_VERSION="88e186e4b9d8f4d5a16af89a68071c0710f119bf"
```

Comment:
please use the 19.0b1-Matrix tag instead of the githash

Reply:
fixed